src/KafkaFlow.Serializer.Json/JsonSchemaRegistrySerializer.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Confluent.Kafka;
+using Confluent.SchemaRegistry;
+using Confluent.SchemaRegistry.Serdes;
+using NJsonSchema.Generation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace KafkaFlow.Serializer.Json
+{
+    /// <summary>
+    /// A json message serializer integrated with the confluent schema registry
+    /// </summary>
+    public class JsonSchemaRegistrySerializer : IMessageSerializer
```

Comment:
On another hand, maybe it could be nice to have explicit 'SchemaRegistry' in the name of the serializers that are integrated with it.

Another suggestion could be ConfluentJsonSchemaRegistrySerializer.

Reply:
I would second @dougolima suggestion to have an explicit reference to 'SchemaRegistry': ConfluentJsonSchemaRegistrySerializer

Reply:
I think the name ConfluentJsonSchemaRegistrySerializer is too big and as we have no option to use or not the Schema Registry in these serializers, I think there is no reason to put the name in there. Just my opinion 😄 

I think the users should know the characteristics of each serializer, we must have some documentation with it. Initially, I thought in something like this

| KafkaFlow Serializer                                                                                                | Serialization                                                                                   | Client Package					          | Scema Registry
| ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------- | ---------------------------------------- | -------------------- |
| [KafkaFlow.Serializer.Json](https://www.nuget.org/packages/KafkaFlow.Serializer.Json/)                             | JSON                                                                                            | System.Text.Json                         | No
| [KafkaFlow.Serializer.ConfluentJson](https://www.nuget.org/packages/KafkaFlow.Serializer.Json/)                    | JSON                                                                                            | Confluent.SchemaRegistry.Serdes.Json     | Yes

Reply:
Yeah, the name is big. However, the benefit is that only the reading the name of the serializer you will know it does.

I agree the clients should know the characteristics of the serializers and we need to have a wiki page about it as well, but I think this doesn't avoid us having a name that would ease the understanding.

IMHO, the pattern could be [**PACKAGE-USED**(Confluent, Core,etc) ][**TYPE OR SERIALIZATION**(json, avro, etc)][**SCHEMAREGISTRY**(or not)]**Serializer**.